src/footer.js

```diff
@@ -0,0 +1,13 @@
+import './header.css';
+
+function Footer() {
+    return (
```

Comment:
```suggestion
const Footer = () => (
```
Please be consistent throughout, preferable to use ES6 standard.